client/context/query.go

```diff
@@ -300,10 +368,15 @@ func (ctx CLIContext) query(path string, key cmn.HexBytes) (res []byte, err erro
 	}
 
 	resp := result.Response
-	if !resp.IsOK() {
```

Comment:
Is `resp.IsOK()` invalid?

Reply:
Sorry. Maybe it is a mistake in code merging. I have changed it to resp.IsOK which seems more elegant